src/classic.js

```diff
@@ -36,8 +32,7 @@ export default class ClassicEditor extends StandardEditor {
 
 		this.data.processor = new HtmlDataProcessor();
 
-		this.ui = new ClassicEditorUI( this );
-		this.ui.view = new BoxedEditorUIView( this.locale );
+		this.ui = new ClassicEditorUIView( this, this.locale );
```

Comment:
I've got mixed feelings. You've got `editor.editing.view` but `editor.view`. But in the first place, I think that we may need to bring back the editor UI controller back. Why do editing and data have controllers and not the UI? Because UI has very little logic now. But removing it just because we don't see it useful yet may be premature and creates a lack of consistency now. Both solutions (keeping empty controller or the lack of consistency) are wrong.